tools/logger/convert.h

```diff
@@ -47,4 +48,6 @@ struct convert_config {
 void log_err(FILE *out_fd, const char *fmt, ...);
 uint32_t get_uuid_key(const struct snd_sof_uids_header *uids_dict,
 		      const struct sof_uuid_entry *entry);
+char *format_uid_raw(const struct sof_uuid_entry *uid_entry, int use_colors,
+		     int name_first);
 int convert(struct convert_config *config);
```

Comment:
Where is the implementation of format_uid_raw? If the implementation is another patch of this series then add this prototype to that patch.

If format_uid_raw already exists in the code, then add the prototype in a separate patch.

Reply:
This function is already implemented in converter.c